dashboard/app/controllers/followers_controller.rb

```diff
@@ -61,8 +61,15 @@ def student_user_new
       redirect_to root_path, notice: I18n.t('follower.registered', section_name: @section.name)
     else
       @user = User.new
-      # if there is no logged in user or no section, render the default student_user_new view which
-      # includes the section code form or sign up form
+
+      # if this is a picture or word section, redirect to the section login page so that the student
+      # does not have to type in the full URL
+      if @section && (@section.login_type == Section::LOGIN_TYPE_PICTURE || @section.login_type == Section::LOGIN_TYPE_WORD)
```

Comment:
nit: My preference is for `[Section::LOGIN_TYPE_PICTURE, Section::LOGIN_TYPE_WORD].include? @section.login_type`.